src/cc_toolchain/wrapper/linux_gcc-gcc

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash --norc
+
+# Copyright 2018 Josh Pieper, jjp@pobox.com.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# we put the `-Wl,--no-as-needed` flag here because it needs to be put before
+# the -l[target] dependencies for correct linking
+exec /usr/bin/x86_64-linux-gnu-gcc -Wl,--no-as-needed "$@"
```

Comment:
Just noticed this because I had to compile on a computer where `gcc-7` was not the default

```
exec /usr/bin/gcc-7 -Wl,--no-as-needed "$@"
```

Reply:
Same sorta thing goes for the other wrappers (`gcc-nm-7` instead of `x86_64-linux-gnu-gcc-nm`, etc.)

Reply:
Actually, please ignore all the above. We probably want to lean towards using the default gcc, rather then a specific version, since we need to compile our code with whatever version was used to compile the 3rd party libraries. Sorry!

Reply:
Just for my understanding, does that mean all this toolchain is really doing is saying "use any version of gcc, but here's 7.0 just in case"?

Reply:
Right now this is saying "use the system default version of gcc". There's definitely some inconsistencies around us saying "gcc 7" when it's really the system version of gcc, but we can clean that up a bit later, don't want to drag this out.